doc/classes/Callable.xml

```diff
@@ -46,6 +46,24 @@
 		    # Prints "Attack!", when the button_pressed signal is emitted.
 		    button_pressed.connect(func(): print("Attack!"))
 		[/codeblock]
+		In GDScript, you can access methods and global functions as [Callable]s using property syntax.
```

Comment:
"Property syntax" or "dot syntax"? Which is it?

Reply:
There is a terrible secret, this works too:

```gdscript
node["queue_free"]
array["clear"]
```

That's why I called it "property syntax" here. Below I mentioned "dot syntax" in the context of `Dictionary` because we had the idea of allowing `dict.clear` as `Callable`, but leaving `dict["clear"]` as a key (but this is confusing and would break compatibility).

Reply:
Yeah, I was aware of this. In that case, if this is what it refers to, you could maybe say "[...]as [Callable] as if they were properties"?

Reply:
I just removed this part, since global functions are not properties.